cartography/intel/aws/ec2/snapshots.py

```diff
@@ -36,8 +37,15 @@ def get_snapshots(boto3_session: boto3.session.Session, region: str, in_use_snap
     self_owned_snapshot_ids = {s['SnapshotId'] for s in snapshots}
     other_snapshot_ids = set(in_use_snapshot_ids) - self_owned_snapshot_ids
     if other_snapshot_ids:
-        for page in paginator.paginate(SnapshotIds=list(other_snapshot_ids)):
-            snapshots.extend(page['Snapshots'])
+        try:
+            for page in paginator.paginate(SnapshotIds=list(other_snapshot_ids)):
+                snapshots.extend(page['Snapshots'])
+        except ClientError as e:
+            if e.response['Error']['Code'] == 'InvalidSnapshot.NotFound':
+                logger.warning(f"Failed retrieve in-use, not owned snapshots. Continuing anyway. Error - {e}")
```

Comment:
Me being nitpicky :)
```suggestion
                logger.warning(f"Failed to retrieve page of in-use, not owned snapshots. Continuing anyway. Error - {e}")
```